### ISSUE TITLE
Not functional with new V5 Websocket built into OBS 28+

### ISSUE THREAD

**Issue by ecarlson88** [Node-JF/tag-obs-websocket#1]
Connects then disconnects over and over. Apparently alot has changed with V5 of the websocket. There is a [Websocket 4.X Compat](https://github.com/obsproject/obs-websocket/releases/tag/4.9.1-compat) plugin to still be able to use older V4 tools while running OBS V28+.

**Comment by anthonyeden** [comment]
Hello. Just wondering if anyone has had a look at the effort involved in upgrading to the v5 Websocket? I'm running on 4.9 currently, but conscious this may not be supported forever in OBS.

**Comment by Node-JF** [comment]
> Hello. Just wondering if anyone has had a look at the effort involved in upgrading to the v5 Websocket? I'm running on 4.9 currently, but conscious this may not be supported forever in OBS.

I've had other priorities seeing as this isn't urgent with the compatibility plugin being available, but I'm aiming to do some work on it this month as things are a bit quieter.

**Comment by anthonyeden** [comment]
Of course, makes sense. Let me know if you'd like me to help with any testing.
